src/content/1.7/release/release-a-native-PS-module.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Release a native PrestaShop module
+weight: 3
+---
+
+# Release a new version of a native PrestaShop module
+
+Some modules are embedded within PrestaShop ZIP archives and loaded through Composer package manager. They consequently are listed in the `composer.json` of the Core project.
+
+This is what we call "native PrestaShop modules". The source code of these modules is hosted on GitHub.
+
+This page explains what is needed to release these modules.
+
+## Requirements
+
+### Mandatory requirements
+
+The module should be valid which means:
+
+- it works as expected
+- it complies with modules [contribution guidelines][contribution-guidelines]: license headers must be valid and necessary files (LICENSE, CONTRIBUTORS) must be embedded
+
+### Recommended requirements
+
+It is recommended:
+
+- to have enabled [Release Drafter](https://github.com/release-drafter/release-drafter) on the module to automatize the process of creating a new GitHub release
+- have enabled some Continous Integration tools such as phpstan or php-cs-fixer on the module
+
+These tools will help releasing a clean module in a smooth manner but the release can happen without them.
+
+## Release process
+
+
+When multiple improvements (bug fixes, enhancements or new features) have been merged into the `dev` branch of a module, we can merge them into `master` to deliver a new release of the module.
+
+### Choosing the new version number
+
+We follow [SemVer](https://semver.org/) guidelines:
+
+If the new version of the module is backward compatible, you can increase the version number either by a minor version or by a patch version.
+
+However, if the new version of the module breaks backward compatibility, it must be a new major version.
+
+### Update version number in the code
+
+Create a Pull Request targeting `dev` branch. In this PR:
+
+Update the module version in the following files:
+
+- `config.xml`
+- `<module_name>.php`
+
+Commit the changes with git message:
+```
+// Version updated to v<new_version>
+```
+Get this PR approved and merge it into `dev`. See an [example](https://github.com/PrestaShop/ps_shoppingcart/pull/50/) of such PR. 
+
+### Merge dev into master
+
+Then create a new PR to merge branch `dev` into branch `master`. See an [example](https://github.com/PrestaShop/ps_shoppingcart/pull/51) of such PR.
```

Comment:
```suggestion
Create a new PR to merge branch `dev` into branch `master`. See an [example](https://github.com/PrestaShop/ps_shoppingcart/pull/51) of such PR.
```

Reply:
Fixed in https://github.com/PrestaShop/docs/pull/591/commits/415b4cd1258516cc371bb750269e85a68546e04b